src/status_im/contexts/chat/group/events.cljs

```diff
@@ -37,3 +39,16 @@
                      :on-success  (fn [response]
                                     (rf/dispatch [:chat-updated response true])
                                     (when on-success (on-success)))}]}))
+
+(rf/reg-event-fx :group-chat/set-manage-members-error
+ (fn [{:keys [db]} [{:keys [chat-id error?]}]]
+   {:db (assoc-in db
+         [:group-chat/manage-members-error chat-id]
+         error?)
+    :fx [(when error?
+           {:dispatch
+            [:toasts/upsert
+             {:type :negative
+              :text (i18n/label :t/new-group-limit
+                                {:max-contacts
+                                 constants/max-group-chat-participants})}]})]}))
```

Comment:
I wonder if this actually works, AFAIK vectors and maps have a lot in common in the Clojure core impl. but here the "official" API uses a vector of vectors:

```clojure
{:fx [ [:dispatch ...] ]

;; you used a map:
{:fx [ {:dispatch ...} ]
```

Reply:
Should be vectors of effects inside `:fx` as you noticed @ulisesmac. We should be consistent as much as possible.

Reply:
`map` also works, but I’ve updated it to `vector` as suggested to maintain consistency.